.github/workflows/ci.yaml

```diff
@@ -85,32 +85,15 @@ jobs:
       - name: Cache Rust artifacts
         uses: actions/cache@v2
         if: |
-          matrix.cmd == 'run-cargo-clippy' || matrix.cmd == 'run-functions-examples --application-variant=rust'
+          matrix.cmd == 'run-cargo-clippy' || matrix.cmd == 'run-examples --application-variant=rust' || matrix.cmd == 'run-examples --application-variant=cpp' || matrix.cmd == 'run-functions-examples --application-variant=rust'
```

Comment:
Just for clarification: are we successively adding commands, i.e., `run-examples for cpp and rust` this time , which use the cache?